core/parachain/candidate_descriptor_v2.hpp

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <boost/endian/conversion.hpp>
+
+#include "crypto/sr25519_provider.hpp"
+#include "parachain/pvf/pvf_error.hpp"
+#include "parachain/transpose_claim_queue.hpp"
+#include "parachain/types.hpp"
+#include "parachain/ump_signal.hpp"
+
+namespace kagome::network {
```

Comment:
If there are corresponding function implementations from Polkadot-SDK used as reference, please add links to them in comments